package.json

```diff
@@ -27,6 +27,7 @@
     "jsonlint": "node scripts/jsonlint.js",
     "lint": "eslint . --cache --report-unused-disable-directives --ext .js,.ts,.tsx",
     "lint:ci": "eslint . --report-unused-disable-directives --ext .js,.ts,.tsx",
+    "perf": "yarn webpack --config test/perf/webpack.config.js --mode \"production\" && concurrently \"npx serve .\" \"npx wait-on http://localhost:5000/test/perf/debug && node scripts/perf.js \"",
```

Comment:
Can we wait in `scripts/perf` instead?

You could also move `serve` into `scripts/perf` so that you can communicate the port that was used. Otherwise this task could fail if `:5000` is already used.

Reply:
Done, serve and wait are in `scripts/perf`